denizen_scripts/hub1/repo-link/Adriftus_Moderator_Panel/panels/mod_actions_inv.dsc

```diff
@@ -0,0 +1,47 @@
+# -- ACTIONS PANEL --
+mod_actions_inv:
+  type: inventory
+  debug: false
+  title: <&6>A<&e>MP <&f>· <&5>Actions on <&d><player.flag[amp_map].as_map.get[uuid].as_player.name>.
+  inventory: CHEST
+  size: 27
+  definitions:
+    x: <item[air]>
+    b1: <item[light_blue_stained_glass_pane].with[display_name=<&r>]>
+    b2: <item[cyan_stained_glass_pane].with[display_name=<&r>]>
+    back: <item[red_stained_glass_pane].with[display_name=<&c><&l>↩<&sp>Player<&sp>list;nbt=<list[to/online]>]>
+    head: <item[mod_player_item]>
+  procedural items:
+    - define muteItem <tern[<player.flag[amp_map].as_map.get[uuid].as_player.has_flag[muted]>].pass[<item[mod_unmute_item]>].fail[<item[mod_mute_item]>]>
+    - define sendItem <item[mod_send_item]>
+    - define kickItem <tern[<player.has_permission[mod.moderator]>].pass[<item[mod_kick_item]>].fail[<item[air]>]>
+    - define banItem <tern[<player.has_permission[mod.moderator]>].pass[<item[mod_ban_item]>].fail[<item[air]>]>
+    - determine <list[<[muteItem]>|<[sendItem]>|<[kickItem]>|<[banItem]>]>
```

Comment:
This doesn't need a list wrapper, you can use `<[muteItem]>|<[sendItem]>|<[kickItem]>|<[banItem]>`